site/docs/view-spec.md

```diff
@@ -0,0 +1,253 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+ 
+# View Support in Iceberg
+
+## Background and Motivation
+Most compute engines (e.g. Trino and Apache Spark ) support logical views, commonly known as ‘views’. A view is a logical table that can be referenced by future queries. Views do not contain any data. Instead, the query stored by the view is executed every time the view is referenced by another query. Views and tables occupy the same namespace.
+Each compute engine stores the metadata of the view in its proprietary format in the metastore of choice. Thus, views created from one engine can not be read or altered easily from another engine even when engines share the metastore as well as the storage system. This document proposes standardizing the view metadata for ease of sharing the views across engines.
+
+### Goals
+* Propose a common metadata format for view metadata, similar to how Iceberg supports a common table format for tables. The common metadata format specification Includes storage format as well as APIs to write/read the metadata
+* Support versioning of views to track how a view evolved over time
+
+## Proposal
```

Comment:
are we going to display this in the website? Should we just make it official documentation for the view spec under the Format section instead of saying it's a proposal? @rdblue

Reply:
Yes, at this point I think we should edit this doc as though it is a pending spec, not a proposal.